student.rb

```diff
@@ -0,0 +1,16 @@
+require './person'
+
+# Student class inherits from Person class
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, name = 'Unknown', parent_permision: true)
+    super(age, name, parent_permision)
+    @classroom = classroom
+    classroom.students << self unless classroom.students.include?(self)
```

Comment:
- [ ] This activity requires that you explicitly define a method that serves as the 'belongs-to' side (_a student belongs to a classroom_), instead of doing same via the constructor. This should be similar to:
```ruby
class Student < Person
  attr_reader :classroom
  .
  .

  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```
You can read further on the same via this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)